blue_ai/envs/transient_goals.py

```diff
@@ -148,31 +179,17 @@ def _gen_mission():
 
     def _gen_grid(self, width, height):
         # Create an empty grid
-        im = self.im
-
         self.grid = Grid(width, height)
-        obj_type = Wall
 
         hasGoal = False
-        hasStart = False
 
-        # Generate the surrounding walls
         for x in range(0, width):
             for y in range(0, height):
-                if np.sum(im[y][x]) == 255:
```

Comment:
This code is probably in need of some improvement, shifting over to using HSL would make a lot of this code more idiomatic and easier to reason with. Putting that on the todo list.